RCAIDE/Library/Components/Fuselages/Fuselage.py

```diff
@@ -143,7 +144,6 @@ def append_fuel_tank(self,fuel_tank):
         return
     
 
-    def compute_fuselage_moment_of_inertia(fuselage, center_of_gravity):
-                    
+    def compute_fuselage_moment_of_inertia(fuselage, center_of_gravity): 

```

Comment:
Can this include self instead of fuselage in the function call? Therefore simplifying the process of calling for fuselage MOI estimate.